client-app/pages/account/dashboard.vue

```diff
@@ -9,7 +9,169 @@
+                  </div>
+                </div>
+              </template>
+              <template #mobile-skeleton>
+                <div
+                  v-for="i of itemsPerPage"
+                  :key="i"
+                  class="grid grid-cols-3 p-6 gap-y-4 gap-x-8 border-b border-gray-200"
+                >
+                  <div class="flex flex-col">
+                    <span class="text-sm text-gray-400">Order number</span>
+                    <div class="h-6 mr-4 bg-gray-200 animate-pulse"></div>
+                  </div>
+
+                  <div class="flex flex-col">
+                    <span class="text-sm text-gray-400">Purchase number</span>
+                    <div class="h-6 mr-4 bg-gray-200 animate-pulse"></div>
+                  </div>
+
+                  <div class="flex flex-col">
+                    <span class="text-sm text-gray-400">Status</span>
+                    <div class="h-6 bg-gray-200 animate-pulse"></div>
+                  </div>
+
+                  <div class="flex flex-col">
+                    <span class="text-sm text-gray-400">Invoice</span>
+                    <div class="h-6 bg-gray-200 animate-pulse"></div>
+                  </div>
+
+                  <div class="flex flex-col">
+                    <span class="text-sm text-gray-400">Date</span>
+                    <div class="h-6 bg-gray-200 animate-pulse"></div>
+                  </div>
+
+                  <div class="flex flex-col">
+                    <span class="text-sm text-gray-400">Total</span>
+                    <div class="h-6 mr-4 bg-gray-200 animate-pulse"></div>
+                  </div>
+                </div>
+              </template>
+              <template #desktop-body>
+                <tr
+                  v-for="order in orders"
+                  :key="order.id"
+                  class="even:bg-gray-50 hover:bg-gray-200 cursor-pointer"
+                  @click="openOrderDetails(order)"
+                >
+                  <td class="p-5 overflow-hidden overflow-ellipsis">
+                    {{ order.number }}
+                  </td>
+                  <td class="p-5 overflow-hidden overflow-ellipsis">
+                    {{ order.purchaseOrderNumber }}
+                  </td>
+                  <td class="p-5 overflow-hidden overflow-ellipsis">{{ order.inPayments?.[0]?.number }}</td>
+                  <td class="p-5 overflow-hidden overflow-ellipsis">
+                    {{ moment(order?.createdDate).format("YYYY-MM-DD") }}
+                  </td>
+                  <td class="p-5 overflow-hidden overflow-ellipsis">
+                    <TableStatusBadge :status="order.status" class="mx-auto"></TableStatusBadge>
+                  </td>
+                  <td class="p-5 overflow-hidden overflow-ellipsis text-right">{{ order.total?.formattedAmount }}</td>
+                </tr>
+              </template>
+              <template #desktop-empty>
+                <tr>
```

Comment:
For what purpose is this empty row?

Reply:
It's the same reason it was in addresses component - workaround to use "colspan" attribute. I'll add a comment on code